File: M3Pact.Web/src/app/shared/models/user-login.ts
import { ClientViewModel } from './DepositLog/client.model';
import { SelectedClients } from '../../admin/users/all-users/models/selected-clients.model';
import { ValidationResponseViewModel } from '../../common/models/validation.model';

export class UserLogin extends ValidationResponseViewModel {
    public userId: any;
    public firstName: string;
    public lastName: string;
    public middleName: string;
    public userName: string;
    public mobileNumber: any;
    public email: string;
    public password: string;
    public recordStatus: string;
    public isActive: boolean;
    public isMeridianUser: boolean;
    public roleCode: string;
    public roleName: string;
    public incorrectLoginAttemptsCount: number;
    public lastSuccessfulLogin: Date;
    public lockoutEndDateUtc: Date;
    public lastPasswordChanged: Date;
    public lockoutEnabled: boolean;
    public clients: ClientViewModel[];
    public selectedClientData: SelectedClients[];
    public forgotPasswordToken: string;
    public loggedInUserID: string;
    constructor() {
        super();
        this.clients = [];
        this.selectedClientData = new Array<SelectedClients>();
    }
}

File: M3Pact.BusinessModel/Admin/UserLogin.cs
using System;
using System.Collections.Generic;

namespace M3Pact.BusinessModel.BusinessModels
{
    public class UserLoginDTO
    {
        public UserLoginDTO()
        {
            Client = new List<ClientDetails>();
        }
        public string UserId { get; set; }
        public string UserName { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string MobileNumber { get; set; }
        public string Email { get; set; }
        public string RoleCode { get; set; }
        public string RoleName { get; set; }
        public bool? IsMeridianUser { get; set; }
        public string Password { get; set; }
        public string ForgotPasswordToken { get; set; }
        public short? IncorrectLoginAttemptsCount { get; set; }
        public DateTime? LastSuccessfulLogin { get; set; }
        public DateTime? LockoutEndDateUtc { get; set; }
        public DateTime? LastPasswordChanged { get; set; }
        public bool? LockoutEnabled { get; set; }
        public string RecordStatus { get; set; }
        public List<ClientDetails> Client { get; set; }
        public string LoggedInUserID { get; set; }
    }
}
